tools/release/image-binary-release/delete-s3-release.sh

```diff
@@ -84,14 +84,14 @@ function delete_s3_objects_from_s3_bucket() {
     )
 
     for i in "${s3_path[@]}"; do
-        s3_latest_key=$(echo "${i}")
-        s3_version_key=$(echo ${s3_latest_key} | sed s/latest/${version}/g)
+        s3_latest_key="${i}"
+        s3_version_key="${s3_latest_key/latest/${version}}"
```

Comment:
Can you explain this change a bit more?

Reply:
https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion

Reply:
This feature: `${parameter/pattern/string}`

Reply:
This was the warning that I was getting: https://www.shellcheck.net/wiki/SC2001

Reply:
```suggestion
        s3_version_key="${s3_latest_key//latest/${version}}"
```
Should it be two slashes here based on the link you provided?